Cesium.Compiler/stdlib/limits.h

```diff
@@ -0,0 +1,34 @@
+#define CHAR_BIT      8
+#define SCHAR_MIN   (-128)
+#define SCHAR_MAX     127
+#define UCHAR_MAX     0xff
+
+#define MB_LEN_MAX    5
+#define SHRT_MIN    (-32768)
+#define SHRT_MAX      32767
+#define USHRT_MAX     0xffff
+#define INT_MIN     (-2147483647 - 1)
+#define INT_MAX       2147483647
+#define UINT_MAX      0xffffffff
+#define LONG_MIN    (-2147483647L - 1)
+#define LONG_MAX      2147483647L
+#define ULONG_MAX     0xffffffffUL
+#define LLONG_MAX     9223372036854775807i64
+#define LLONG_MIN   (-9223372036854775807i64 - 1)
+#define ULLONG_MAX    0xffffffffffffffffui64
```

Comment:
Sorry, what's `i64`? Is it a valid C17 syntax?

Reply:
It seems MSVC is too permissive in what it allows. Will read docs and change accordingly.